diagnostic.rb

```diff
@@ -10,7 +12,12 @@
 end
 
 # question 4:
+Licensee.where hair_color: %w(Hazel), eye_color: %(Black Brown)
 
 # question 5:
+Licensee.where surname: %w(Escobar Whaley) given_name: %w(Marilynn Chris)
```

Comment:
Not quite. This `where` would also grab someone with the names "Chris Escobar" or "Marilynn Whaley", and we don't want that. You could either write a `.where` that only grabs "Marilynn Escobar" or "Chris Whaley", or access Licensee twice.